BUILDING.md

```diff
@@ -138,13 +140,25 @@ for more information.
 Note that the above requires that `python` resolve to Python 2.6 or 2.7
 and not a newer version.
 
-To run the tests:
+#### Running Tests
+
+To verify the build:
+
+```console
+$ make test-only
+```
+
+At this point you are ready to make code changes and re-run the tests.
+
+If you are running tests prior to submitting a Pull Request, the recommended
+command is:
 
 ```console
 $ make test
 ```
 
-At this point you are ready to make code changes and re-run the tests!
+Which also runs linters and the tests on the documentation.
```

Comment:
Nit: Instead of `Which also runs linters...` maybe ``` `make test` will run linters... ```

Reply:
I have changed this to

```
`make test` does a full test on the codebase, including linting and
documentation tests.
```

so it's clearer about what it does.